### PR TITLE
Create smart-contract-wallets.md

### PR DESCRIPTION
**Pull request by CryptoManion**
New article to add

### CODE REVIEW

File: docs/wallets/smart-contract-wallets.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+These cutting-edge wallets leverage account abstraction to integrate essential functions into smart contracts, offering users numerous advantages such as bundled transactions, customizable recovery options, and greater privacy in identity management. In this article, we delve into the mechanics of **smart contract wallets**, the **Ethereum account mode**l, and the **EIP 4337 proposal**, exploring how these innovations have the potential to revolutionize the way users interact with cryptocurrencies and blockchain technology.
+
+## What are smart contract wallets? 
+
+To understand smart contract wallets, we need to first explain what are **Externally Owned Accounts (EOAs)**
+
+Externally Owned Accounts (EOAs) are accounts on the [**Ethereum blockchain**](cryptocurrency/ethereum-glossary.md) that are owned and controlled by an end user, as opposed to a [**smart contract**](defi/smart-contracts.md). EOAs are typically created using a [**software wallet** or a **hardware wallet**](wallets/crypto-walets.md), and can be used to send and receive tokens and interact with smart contracts.
```
Comment (yurikoinaba) [inline docs/wallets/smart-contract-wallets.md:24]:
```suggestion
Externally Owned Accounts (EOAs) are accounts on the [**Ethereum blockchain**](cryptocurrency/ethereum-glossary.md) that are owned and controlled by an end user, as opposed to a [**smart contract**](defi/smart-contracts.md). EOAs are typically created using a [**software wallet** or a **hardware wallet**](wallets/crypto-wallets.md), and can be used to send and receive tokens and interact with smart contracts.
```

There is a typo

File: docs/wallets/smart-contract-wallets.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+They facilitate multi-operations or atomic operations, batch multiple user operations into a single transaction, and introduce the concept of a paymaster, allowing users to pay gas fees with their preferred token or even off-chain with a credit card.
+
+EIP 4337 proposes a new object called **user operation**. User operations are created, signed by users, and sent to a **user operation mempool**. A **bundler** (a node or network participant that bundles transactions together and submits them to the blockchain for processing) listens to this mempool, picks up new user operations, and bundles them into a **single entity** called a **bundle transaction**, which is then included in an Ethereum block like any other transaction.
+The bundling process involves **two smart contracts**: the wallet smart contract and a special entry point contract. The entry point contract contains most of the logic for user operation validation and execution.
+
+![smart-contract-wallets](ZBF_graphics_May_ERC_4337.jpg)
```
Comment (yurikoinaba) [inline docs/wallets/smart-contract-wallets.md:51]: `ZBF_graphics_May_ERC_4337` this file hasn't been added

Comment (CryptoManion) [reply docs/wallets/smart-contract-wallets.md:51]: Oh yes, because the folder doesn't exist

Comment (CryptoManion) [reply docs/wallets/smart-contract-wallets.md:51]: I think I added it?

**Comment by CryptoManion** [comment]
PR in july_13 instead

**Pull request closed without merging**